### ISSUE TITLE
Allow taxonomic units to record additional information about specimens

### ISSUE THREAD

**Issue by gaurav** [phyloref/curation-tool#20]
There is a clear difference between a specifier that points to a scientific name and one that points to a specimen that has been identified to a scientific name.

We would like to extend this to document all the information included about the specimen in the paper, such as scientific name, GenBank identifiers to particular genes, locality information and so on. Note that we can use an external identifier to identify a specimen when the specimen is the taxonomic unit, but in some cases, we will want to use a URI to clarify which specimens are included in a taxonomic unit circumscribed by a taxon concept or taxonomic name.

Also add support for specimens that are included solely on the basis of a genetic sequence, i.e. make sure we can include a sequence itself if necessary. This might be useful in environmental genetic surveys or bacterial phylogenies.

**Comment by gaurav** [comment]
I don't think we need this any more -- most phyloreferences use a phrasing like "the type specimen of [species name]" rather than a specimen identifier, and we haven't seen any references to GenBank IDs or locality information in a clade definition yet. Once we get to environmental genetic surveys or bacterial phylogenies, we can consider other forms of specifiers than taxa and specimens or the storing of additional information in the Curation Tool. But for now, I'm going to close this until we have a clear use-case for developing this.

**Comment by hlapp** [comment]
I'd suggest that closing an issue be kept to have relatively clear and simple semantics. Usually this means "it's done", either in the sense of work having been done that addresses the issue sufficiently well, or a conclusion having been made that it is not necessary to address it, or that it is poorly scoped (needing either to be split into, or to be combined with other issues).

I would argue that _deferring work_ is different from that, in the sense that for work that's _deferred_ we assume we will come back to it in the future, only when in the future hasn't been decided yet (and may not be decided for a while). I.e., _deferred work_ is work that we continue to find worth doing. In contrast, closed issues shouldn't need to be revisited other than perhaps for the purposes of annual reporting, because they're _done_.

Your explanation for closing sounds to be like an argument for deferring. Perhaps I misread that, but if that's indeed what you meant, let's not close this but instead use labels (and/or possibly a pseudo-milestone) to mark deferred issues.

**Comment by gaurav** [comment]
Sounds good! I'd classify it as "deferred indefinitely" -- i.e. after thinking this through, I can't imagine that it will ever be in scope for the Curation Tool. But it definitely doesn't meet your criteria of "it's done" (and I could be wrong!), so I'll reopen it and tag it as "wontfix" instead. Will that work, or should we have a new tag specifically for "deferred indefinitely"?
